### DISCUSSION TITLE
지역 필터링은 어떻게 구현하면 좋을까요?

### DISCUSSION THREAD

**Discussion by soeunnPark** [Kernel360/E2E2-CATCHLINE#19] [Ideas]
지도 API에서 위도와 경도를 가져와서 이를 서울시, 송파구 등으로 필터링하고 싶습니다. 어떻게 구현할 수 있을까요?

**Comment by sangwonsheep** [comment]
https://devtalk.kakao.com/t/topic/113464
참고해보면 좋을 것 같아요!
